### PR TITLE
created sidebar and scrollbar

### PR DESCRIPTION
**Pull request by vaishali9080**
merge to main file

### CODE REVIEW

**Comment by vaishali9080** [comment]
@Ruler45

**Comment by Ruler45** [comment]
> merge to main file



> merge to main file

Description must be something appealing

File: src/app/layout.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import "./globals.scss";
-
+import Sidebar from "./components/sidebar";
+import "./components/Scrollbar.css";
```
Comment (Ruler45) [inline src/app/layout.js]: No body should import any css file in layout file except the globals.scss

File: src/app/page.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ export default function Home() {
         width: "500px",
       }}
     >
-      <h1>Home page</h1>
```
Comment (Ruler45) [inline src/app/page.js:16]: Don't make unnecessary changes like this.

**Review by Ruler45 — APPROVED** [review]
